store/postgres/tests/store.rs

```diff
@@ -32,15 +32,16 @@ fn postgres_test_url() -> String {
 }
 
 lazy_static! {
+    static ref LOGGER:Logger = {match env::var_os("GRAPH_LOG") {
```

Comment:
Something's up with the formatting here. It's only tests, so it may not affect the `rustfmt` check on Travis?

Reply:
Yeah, that's pretty strange, and neither `cargo fmt` nor `rustfmt` on the file itself seem to have an opinion on it. I fixed it manually (and rebased to latest master)

Reply:
👍